src/Proxy/AbstractProxy.php

```diff
@@ -204,9 +194,9 @@ protected function prepareArgsLine(ReflectionFunctionAbstract $functionLike)
      *
      * @return string
      */
-    protected function getOverriddenFunction(ReflectionFunctionAbstract $functionLike, $body)
+    protected function getOverriddenFunction(ReflectionFunctionAbstract $functionLike, string $body): string
     {
-        $reflectionReturnType = PHP_VERSION_ID >= 70000 ? $functionLike->getReturnType() : '';
+        $reflectionReturnType = $functionLike->getReturnType();
         $modifiersLine        = '';
         if ($reflectionReturnType) {
             $nullablePrefix = $reflectionReturnType->allowsNull() ? '?' : '';
```

Comment:
Note: this function uses built in function `join` which is alias for `implode` and as such not quite safe to use. Perhaps you should use this opportunity to replace it with `implode`?